src/utilities/process-reply.js

```diff
@@ -53,7 +53,7 @@ module.exports = async function processReply({
           const formattedCode = formatter(content, supportedLanguage);
 
           ref.channel.send(
-            formatMessageWithCodeblock(languageGuesses[1], formattedCode)
+            formatMessageWithCodeblock(supportedLanguage, formattedCode)
```

Comment:
This no longer seems possible, if we are in the `else` case but the `if` case is if there is a `supportedLanguage` in the first place?